tests/e2e/specs/setup-mc/step-3-confirm-store-requirements.test.js

```diff
@@ -284,25 +284,25 @@ test.describe( 'Confirm store requirements', () => {
 				} );
 
 				test( 'should see all checkboxes are not checked', async () => {
-					const prelaunchChecklistCheckboxes =
+					const checkboxes =
 						storeRequirements.getPrelaunchChecklistCheckboxes();
 
-					for ( const checkbox of await prelaunchChecklistCheckboxes.all() ) {
+					await expect( checkboxes ).toHaveCount( 5 );
+
+					for ( const checkbox of await checkboxes.all() ) {
 						await expect( checkbox ).not.toBeChecked();
 					}
 				} );
 
 				test( 'should toggle all checklists and see the "Confirm" button in each checklist', async () => {
-					const prelaunchChecklistToggles =
-						storeRequirements.getPrelaunchChecklistToggles();
```

Comment:
I'd remove `getPrelaunchChecklistToggles`
https://github.com/woocommerce/google-listings-and-ads/blob/dev/fix-e2e-network-issues/tests/e2e/utils/pages/setup-mc/step-3-confirm-store-requirements.js#L197:L206

Or do you envision using that function later?

Reply:
Thanks! Removed in https://github.com/woocommerce/google-listings-and-ads/pull/2116/commits/4ebc91d6c0bc91cc321e00b59927d66f0cc57920.